.husky/pre-push

```diff
@@ -12,9 +12,4 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-. "$(dirname "$0")/_/husky.sh"
-
-npm run check
```

Comment:
Can you just comment these out with an additional comment to explain why they are disabled?

Reply:
Done